src/entry-points/content/AllMediaElementsController.ts

```diff
@@ -57,7 +57,11 @@ export type TelemetryMessage =
     elementLikelyCorsRestricted: boolean,
     elementCurrentSrc?: string,
     createMediaElementSourceCalledForElement: boolean,
-    elementRemainingDuration: number,
+     /**
+     * Remember that this could be `Infinity` for live streams,
```

Comment:
For some reason (at least on twitch.tv, Firefox developer 134.0b5, Fedora 40), live streams have just an unusually large duration, but don't show up as Infinity.
Example: 
![image](https://github.com/user-attachments/assets/574b5bc1-b933-4880-9e38-b731c030e829)
![image](https://github.com/user-attachments/assets/abb95d9c-faeb-47bf-94e9-d987e64a1244)